integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/PersonFhirResourceProviderIntegrationTest.java

```diff
@@ -0,0 +1,455 @@
+		Person reReadPerson = readResponse(response);
+		
+		assertThat(reReadPerson.getBirthDate(), equalTo(birthDate));
+	}
+	
+	@Test
+	public void shouldReturnBadRequestWhenDocumentIdDoesNotMatchPersonIdAsXML() throws Exception {
+		// get the existing record
+		MockHttpServletResponse response = get("/Person/" + PERSON_UUID).accept(FhirMediaTypes.XML).go();
+		Person person = readResponse(response);
+		
+		// update the existing record
+		person.setId(WRONG_PERSON_UUID);
+		
+		// send the update to the server
+		response = put("/Person/" + PERSON_UUID).xmlContext(toXML(person)).go();
+		
+		assertThat(response, isBadRequest());
+		assertThat(response.getContentType(), is(FhirMediaTypes.XML.toString()));
+		assertThat(response.getContentAsString(), notNullValue());
+		
+		OperationOutcome operationOutcome = readOperationOutcome(response);
+		
+		assertThat(operationOutcome, notNullValue());
+		assertThat(operationOutcome.hasIssue(), is(true));
+	}
+	
+	@Test
+	public void shouldDeleteExistingPerson() throws Exception {
+		MockHttpServletResponse response = delete("/Person/" + PERSON_UUID).accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, isOk());
+		
+		response = get("/Person/" + PERSON_UUID).accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, statusEquals(HttpStatus.GONE));
+	}
+	
+	@Test
+	public void shouldReturnNotFoundWhenDeletingNonExistentPerson() throws Exception {
+		MockHttpServletResponse response = delete("/Person/" + WRONG_PERSON_UUID).accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, isNotFound());
+		assertThat(response.getContentType(), is(FhirMediaTypes.JSON.toString()));
+		assertThat(response.getContentAsString(), notNullValue());
+		
+		OperationOutcome operationOutcome = readOperationOutcome(response);
+		
+		assertThat(operationOutcome, notNullValue());
+		assertThat(operationOutcome.hasIssue(), is(true));
+	}
+	
+	@Test
+	public void shouldSearchForAllPersonsAsJson() throws Exception {
+		MockHttpServletResponse response = get("/Person").accept(FhirMediaTypes.JSON).go();
+		
+		assertThat(response, isOk());
+		assertThat(response.getContentType(), is(FhirMediaTypes.JSON.toString()));
+		assertThat(response.getContentAsString(), notNullValue());
+		
+		Bundle results = readBundleResponse(response);
+		
+		assertThat(results, notNullValue());
+		assertThat(results.getType(), equalTo(Bundle.BundleType.SEARCHSET));
+		assertThat(results.hasEntry(), is(true));
+		
+		List<Bundle.BundleEntryComponent> entries = results.getEntry();
+		
+		assertThat(entries, everyItem(hasProperty("fullUrl", startsWith("http://localhost/ws/fhir2/R3/Person/"))));
+		assertThat(entries, everyItem(hasResource(instanceOf(Person.class))));
+		assertThat(entries, everyItem(hasResource(validResource())));
+	}
+	
+	@Test
+	public void shouldReturnSortedAndFilteredSearchResultsForPersonsAsJson() throws Exception {
+		MockHttpServletResponse response = get("/Person?family=Doe&_sort=given").accept(FhirMediaTypes.JSON).go();
```

Comment:
You get an error for this test because "family" is not a configured endpoint for the Person resource. You might as well try:
get("/Person?name=Doe&_sort=given")